server/webhook_http_test.go

```diff
@@ -643,19 +662,35 @@ func TestWebhookHTTP(t *testing.T) {
 				TeamId: "theteamid",
 			}, (*model.AppError)(nil))
 
+			api.On("GetDirectChannel",
+				mock.AnythingOfType("string"),
+				mock.AnythingOfType("string")).
+				Return(&model.Channel{}, (*model.AppError)(nil))
+
 			p := Plugin{}
+
 			p.updateConfig(func(conf *config) {
 				conf.Secret = validConfiguration.Secret
 			})
 			p.SetAPI(api)
 
 			if tc.CurrentInstance {
-				p.currentInstanceStore = mockCurrentInstanceStore{&p}
+				p.currentInstanceStore = mockCurrentInstanceStore{plugin: &p}
 			} else {
 				p.currentInstanceStore = mockCurrentInstanceStoreNoInstance{&p}
 			}
+			s := startServer(t)
```

Comment:
See my previous comment, you should be able to simplify all this if you start a default (started) test server